theme/templates/base.html

```diff
@@ -23,25 +23,29 @@
     </head>
     <body class="{% if page and page.body_class %}{{ page.body_class }}{% endif %}">
         <!-- Nav -->
-        <nav class="col-12 primary">
-            <a href="/">
-                <img src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/images/kiwi_logo_h50.png" name="logo">
-            </a>
-
-            <ul class="topnav">
-                <li class="right">
-                    <form action="http://google.com/search" method="get">
-                        <input name="sitesearch" value="http://kiwitcms.org" type="hidden">
-                        <input class="search" name="q" placeholder="Search" type="text">
-                    </form>
-                </li>
-
-                <li class="right"><a href="{{ SITEURL }}/blog/">Blog</a></li>
-                <li class="right"><a href="http://mrsenko.com">Support</a></li>
-                <li class="right"><a href="https://github.com/kiwitcms/Kiwi">Community</a></li>
-                <li class="right"><a href="http://kiwitcms.readthedocs.io">Documentation</a></li>
-                <li class="right"><a href="https://demo.kiwitcms.org">Checkout Demo</a></li>
-            </ul>
+        <nav class="col-12 topnav primary">
+            <div class="col-4">
+                <a href="/">
+                    <img src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/images/kiwi_logo_h50.png" name="logo">
+                </a>
+            </div>
+            <button class="open-menu accent-unimportant"><i class="fa fa-align-justify" aria-hidden="true"></i></button>
+            <div class="col-8 mobilenav">
+                <button class="close-menu accent-unimportant"><i class="fa fa-times" aria-hidden="true"></i></button>
+                <ul>
+                    <li class="right"><a href="https://demo.kiwitcms.org">Checkout Demo</a></li>
+                    <li class="right"><a href="http://kiwitcms.readthedocs.io">Documentation</a></li>
+                    <li class="right"><a href="https://github.com/kiwitcms/Kiwi">Community</a></li>
+                    <li class="right"><a href="http://mrsenko.com">Support</a></li>
+                    <li class="right"><a href="{{ SITEURL }}/blog/">Blog</a></li>
+                    <li class="right">
+                        <form action="http://google.com/search" method="get">
+                            <input name="sitesearch" value="http://kiwitcms.org" type="hidden">
+                            <input class="search" name="q" placeholder="Search" type="text">
+                        </form>
+                    </li>
+                </ul>
+            </div>
```

Comment:
Revert this entire section to what it was before and add the open-menu and close-menu buttons.
You will only need an id for the div containing the ul element so create one with a meaningful name.

Because of the change that will be reverted some style definitions need to be removed as well, orthers will need to change selectors.